CHANGELOG.md

```diff
@@ -9,6 +9,12 @@ All notable changes to this project will be documented in this file.
 - Active Directory's `samAccountName` generation can now be customized ([#454]).
 - Added experimental cert-manager backend ([#482]).
 
+### Changed
+
+- Refactored hostname validation ([#494]).
+  - BREAKING: Hostname validation is now somewhat stricter.
+  - BREAKING: Hostname validation is now enforced in CRD.
```

Comment:
@sbernauer This regressed in upstream op-rs (https://github.com/stackabletech/operator-rs/pull/863/files/87ca3050bda8070d36076706bbea41f2ee008e3d#diff-5ca013a250a561e7d4a42a3272200ba0dc20b0ad8345fb7660c904de3407f791).

Reply:
Yep, we had some discussions around RFC 1123 and renamed it accordingly to the RFC (at least we hope ^^), before it gaines widespread adoption. We would have involved you but you where on vacation and we did not want to further delay the PR, but please feel free to veto, happy to discuss with @Techassi!